pkg/rebuild/rebuild/rebuildremote.go

```diff
@@ -99,10 +100,16 @@ var standardBuildTpl = template.Must(
 	).Parse(
 		textwrap.Dedent(`
 				set -eux
+				{{- if .EnableSyscallMonitor}}
+				docker run --name=tetragon --pid=host --cgroupns=host --privileged -v=/sys/kernel/btf/vmlinux:/var/lib/tetragon/btf quay.io/cilium/tetragon:v1.1.2 /usr/bin/tetragon --export-filename=/workspace/tetragon.jsonl
+				{{- end}}
 				cat <<'EOS' | docker buildx build --tag=img -
 				{{.Dockerfile}}
 				EOS
 				docker run --name=container img
+				{{- if .EnableSyscallMonitor}}
+				docker kill tetragon
```

Comment:
I seem to remeber this coming up earlier but do we need this? Will the log be left in a coherent state and be up-to-date if the process remains active? It seems more signifcant in the netproxy case where we want to exclude the curl call but here seems less clear-cut.

Reply:
It was definitely necessary when these were GCB steps because GCB would wait for the tetragon step to finish.

I still think it's worth keeping for a similar reason to the netproxy, in that we don't want to add noise in the syscall log from the rest of the execution afterwards.